scripts/create_test_forms.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+'''Generate test form entries/items as JSON to use for load testing.
+'''
+
+import json
+import csv
+import uuid
+
+NUM_TEST_FORMS = 100
+
+def get_uuid():
+  return 'test' + str(uuid.uuid4())
+
+def generate_test_form_data(num_test_forms):
+  uuids = []
+  for i in range(num_test_forms):
+    uuids.append(get_uuid())
+  return uuids
+
+def generate_test_form_json(test_form_data):
+  forms = []
+  for uuid in test_form_data:
+      # 1. id must be unique because the forms container partitions on it (/id)
+      # 2. authToken must be unique because our code looks for a unique authtoken
+      forms.append({
+          'id': uuid,
+          'authToken': uuid,
```

Comment:
I think this is fine, but why not just number them so it's easier for them to script? E.g., test0, test1, ...

Reply:
haha good point, i originally just had uuids, then prepended 'test' later on, but i could have just numbered them :P 

but on further reflection, i think i'll leave them as uuids since that matches the real payload size for both id and authToken much better than test0/test1, and it seems like the throughput in cosmosdb is hugely influenced by the per-item size...

Reply:
Yeah, now that I know JMeter can parameterize using a csv file, it doesn't seem to matter that much.